roles/ssp/tasks/configure-common.yml

```diff
@@ -26,6 +26,41 @@
   tags:
     - ssp:config:authsources
 
+
+
+
+- name: Copy SP SSL certificate (.crt)
+  copy:
+    content: '{{ item.sp_ssl_certificate }}'
+    dest: "{{ ssp_certdir }}/sp-sso.crt"
+    owner: "{{ ssp_webuser }}"
+    group: "{{ ssp_webgroup }}"
+    mode: "0644"
+  with_items: "{{ ssp_authsources_saml }}"
+  when: item.generate_ssl_certificate|bool == False
+  become: yes
+  no_log: yes
```

Comment:
`no_log: yes` can be omitted here since this task is copying the public certificate